pcsx2/Docs/PCSX2_FAQ.md

```diff
@@ -153,29 +153,29 @@ Gamefixes are not FPS boosters; more often than not, they will degrade performan
 
 *Bug reports of issues caused by speedhacks will NOT be accepted, and will be immediately closed as invalid.*
 
-### Question 25: What are all these EE/IOP and VU options
+### Question 25: What are all these EE/IOP and VU options?
 The PS2 EE, IOP and VU processors are substantially different from a PC CPU and require different rounding and clamping modes to do math accurately. Most games work fine on the default options, but some games might need a different setting. You can check the [PCSX2 Wiki](https://wiki.pcsx2.net/Category:Games) to see if your game needs an alternate setting, or check the [PCSX2 Forums](https://forums.pcsx2.net/) to see if anyone else has posted about it there. 
 
-### Question 26: What are PCSX2 plugins
-PCSX2 uses a plugin framework for various sections of the emulator. A plugin is a small, incomplete piece of software that, when plugged in to another piece of software, provides some sort of additional function. PCSX2 uses plugins for:
+### Question 26: What are PCSX2 plugins?
+Older versions of PCSX2 used a plugin framework for various sections of the emulator. A plugin is a small, incomplete piece of software that, when plugged in to another piece of software, provides some sort of additional function. PCSX2 used to use plugins for:
 *   Video (GSdx)
 *   Controls (Lilypad for Windows, OnePad for Linux)
 *   USB (No plugin included)
 
-Each plugin will have its own Plugin Settings menu, and can be found under the Config menu.
+In newer versions, these have been merged into the main emulator, and there is no longer a way to load external plugins.
 
-### Question 27: Why is my sound garbled up
+### Question 27: Why is my sound garbled up?
 PCSX2's default audio playback mode is Timestretch, which will attempt to stretch sounds out when the emulator is unable to play at full speed, to fill the gaps in the sound playback and prevent even harsher stutters. If you are unable to get the game running at full speed or an acceptable speed where timestretching does not ruin the sound, you can try changing from Timestretch to Async, but this may break games.
 
-### Question 28: Is my controller supported
-For Windows users running LilyPad, PCSX2 will ship with XInput bindings already created. This will automatically work with an Xbox One controller or any other XInput controller. Bindings can easily be made for DirectInput controllers, such as Xbox 360 controllers. PS3, PS4, or Switch Pro controllers should be set up using a compatibility software, such as Steam's Big Picture mode. 
+### Question 28: Is my controller supported?
+For Windows users, PCSX2 will ship with XInput bindings already created. This will automatically work with an Xbox One controller or any other XInput controller. Bindings can easily be made for DirectInput controllers, such as Xbox 360 controllers. PS3, PS4, or Switch Pro controllers should be set up using a compatibility software, such as Steam's Big Picture mode. 
```

Comment:
"Bindings can easily be made for DirectInput controllers"
Should probably be
"Bindings can easily be made for DirectInput and XInput controllers"

Reply:
Went ahead and updated it to mention both.